redhat/lyrionmusicserver.spec

```diff
@@ -460,35 +412,35 @@ function migrateSqueezeboxServerConfig {
 
       echo "WARNING, failed migrating old configuration. You will need to migrate it manually or configure Lyrion Music Server from scratch."
       # Restore the safety copy
-      rm -f -r /var/lib/%{shortname} >/dev/null 2>&1 || :
-      mv /var/lib/%{shortname}.bck /var/lib/%{shortname} >/dev/null 2>&1 || :
+      /usr/bin/rm -f -r /var/lib/%{shortname} >/dev/null 2>&1 || :
+      /usr/bin/mv /var/lib/%{shortname}.bck /var/lib/%{shortname} >/dev/null 2>&1 || :
       return 1
 
    else
 
-      if ! /usr/bin/find /var/lib/%{shortname} -type f -name "*.prefs" -exec sed -i 's#/squeezeboxserver#/%{shortname}#g' {} \; >/dev/null 2>&1; then
+      if ! /usr/bin/find /var/lib/%{shortname} -type f -name "*.prefs" -exec /usr/bin/perl -i.pre-squeeze-to-lyrion -pe "s/\/squeezeboxserver/\/lyrionmusicserver/" {} \; >/dev/null 2>&1; then
```

Comment:
I think it was clearer to use `#` (or whatever character) as the regular expression delimiter, rather than ending up with `/\/`.

Is it correct that the `/g` has been dropped?

Is there a reason to hard-code `lyrionmusicserver` here, when `%{shortname}` is used everywhere else?

Reply:
I had issues when I tested it using # as a delimiter. I'll check again and come back about this.

Reply:
and it is only oversight on my behalf that %{shortname} isn't used.